iothub_client/samples/iothub_ll_client_shared_sample/iothub_ll_client_shared_sample.c

```diff
@@ -187,105 +203,125 @@ int main(void)
-            //bool traceOn = true;
-            //IoTHubDeviceClient_LL_SetOption(device_ll_handle1, OPTION_LOG_TRACE, &traceOn);
-            //IoTHubDeviceClient_LL_SetOption(device_ll_handle2, OPTION_LOG_TRACE, &traceOn);
-#ifdef SET_TRUSTED_CERT_IN_SAMPLES
-            // Setting the Trusted Certificate. This is only necessary on systems without
-            // built in certificate stores.
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle1, OPTION_TRUSTED_CERT, certificates);
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle2, OPTION_TRUSTED_CERT, certificates);
-#endif // SET_TRUSTED_CERT_IN_SAMPLES
-
-#ifdef SAMPLE_HTTP
-            unsigned int timeout = 241000;
-            // Because it can poll "after 9 seconds" polls will happen effectively // at ~10 seconds.
-            // Note that for scalabilty, the default value of minimumPollingTime
-            // is 25 minutes. For more information, see:
-            // https://azure.microsoft.com/documentation/articles/iot-hub-devguide/#messaging
-            unsigned int minimumPollingTime = 9;
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle1, OPTION_MIN_POLLING_TIME, &minimumPollingTime);
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle1, OPTION_HTTP_TIMEOUT, &timeout);
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle2, OPTION_MIN_POLLING_TIME, &minimumPollingTime);
-            IoTHubDeviceClient_LL_SetOption(device_ll_handle2, OPTION_HTTP_TIMEOUT, &timeout);
-#endif // SAMPLE_HTTP
+            IOTHUB_CLIENT_DEVICE_CONFIG config = { 0 };
+            config.deviceId = g_device_states[i].credential->id;
+            config.deviceKey = g_device_states[i].credential->key;
+            config.deviceSasToken = NULL;
+            config.protocol = protocol;
+            config.transportHandle = IoTHubTransport_GetLLTransport(transport_handle);
+
+            if ((g_device_states[i].client_handle = IoTHubDeviceClient_LL_CreateWithTransport(&config)) == NULL)
+            {
+                (void)printf("ERROR: iotHubClientHandle is NULL (%d)!\r\n", i);
+                setup_ok = false;
+            }
+            else
+            {            
+                bool traceOn = true;
+                
+    #ifdef SAMPLE_HTTP
+                unsigned int timeout = 241000;
+                // Because it can poll "after 9 seconds" polls will happen effectively // at ~10 seconds.
+                // Note that for scalabilty, the default value of minimumPollingTime
+                // is 25 minutes. For more information, see:
+                // https://azure.microsoft.com/documentation/articles/iot-hub-devguide/#messaging
+                unsigned int minimumPollingTime = 9;
+    #endif
+                
+                if (IoTHubDeviceClient_LL_SetOption(g_device_states[i].client_handle, OPTION_LOG_TRACE, &traceOn) != IOTHUB_CLIENT_OK)
+                {
+                    (void)printf("Failed setting log tracing (%d)!\r\n", i);
```

Comment:
would it be more info to print the config.deviceId instead of the iteration?

Reply:
yeah, it would. I'll change it to that.